src/main/java/com/cloudblue/connect/internal/operations/ListCaseOperation.java

```diff
@@ -0,0 +1,40 @@
+package com.cloudblue.connect.internal.operations;
+
+import com.cloudblue.connect.api.clients.Client;
+import com.cloudblue.connect.api.exceptions.CBCException;
+import com.cloudblue.connect.api.models.CBCCase;
+import com.cloudblue.connect.api.parameters.filters.Filter;
+import com.cloudblue.connect.api.parameters.filters.OrderBy;
+import com.cloudblue.connect.internal.operations.connections.CBCConnection;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import org.mule.runtime.extension.api.annotation.param.Connection;
+import org.mule.runtime.extension.api.annotation.param.MediaType;
+import org.mule.runtime.extension.api.annotation.param.Optional;
+import org.mule.runtime.extension.api.annotation.param.Parameter;
+import org.mule.runtime.extension.api.annotation.param.display.DisplayName;
+
+import static org.mule.runtime.extension.api.annotation.param.MediaType.ANY;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ListCaseOperation extends BaseListOperation {
+
+    @MediaType(value = ANY, strict = false)
+    @DisplayName("List Helpdesk Cases")
+    public List<CBCCase> listCases(
+            @Connection CBCConnection connection
+    ) throws CBCException {
+
+        Client.Q q = connection
+                .newQ(new TypeReference<ArrayList<CBCCase>>() {})
+                .collection("helpdesk/cases");
```

Comment:
.collection("helpdesk").collection("cases")

Please use the similar approach in all placess.